tests/integration/test_postgresql_protocol/configs/config.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+ <yandex>
+     <logger>
+         <level>trace</level>
+         <log>/var/log/clickhouse-server/clickhouse-server.log</log>
+         <errorlog>/var/log/clickhouse-server/clickhouse-server.err.log</errorlog>
+         <size>1000M</size>
+         <count>10</count>
+     </logger>
+
+     <!-- Used with https_port and tcp_port_secure. Full ssl options list: https://github.com/ClickHouse-Extras/poco/blob/master/NetSSL_OpenSSL/include/Poco/Net/SSLManager.h#L71 -->
+     <openSSL>
```

Comment:
ClickHouse can be built without SSL support (i.e. with `ENABLE_SSL=0`). Is PostgreSQL protocol able to work in such build?

Reply:
Yes, it should work without SSL. I will check build without it.